src/codex.ts

```diff
@@ -81,9 +82,14 @@ export default class EditorJS {
   public exportAPI(editor: Core): void {
     const fieldsToExport = [ 'configuration' ];
     const destroy = (): void => {
-      editor.moduleInstances.Listeners.removeAll();
-      editor.moduleInstances.UI.destroy();
-      editor.moduleInstances.ModificationsObserver.destroy();
+      for(const moduleName in editor.moduleInstances) {
+        if (Object.prototype.hasOwnProperty.call(editor.moduleInstances, moduleName)) {
+          const moduleInstance = editor.moduleInstances[moduleName];
+          if(moduleInstance.destroy && _.isFunction(moduleInstance.destroy)) {
```

Comment:
I think its better to add check for `undefined` inside the `isFunction`

Reply:
Will remove the 'moduleInstance.destroy' check from the if condition. Is a separate check in isFunction required? I can leave the isFunction code as it is and the typeof check should take care even if I pass undefined?